node_modules/gh-orgunit/tests/test-series.js

```diff
@@ -313,6 +315,55 @@ describe('Organisational units - Series', function() {
                 });
             });
         });
+
+        /**
+         * Test that verifies that deleting a series from an organisational unit removes the series from users their calendar
+         */
+        it('verify deleting a series from an organisational unit removes the series from users their calendar', function(callback) {
+            TestsUtil.generateTestUsers(global.tests.apps.cam2013, 2, false, function(simon, nico) {
+
+                // Create an organisational unit with two series under it
+                OrgUnitTestsUtil.assertCreateOrgUnit(global.tests.admins.cam2013.client, 'Test org unit', 'module', null, function(orgUnit) {
+                    var calendarStart = moment().subtract(1, 'day').format();
+                    var calendarEnd = moment().add(30, 'day').format();
+                    SeriesTestsUtil.generateSerieWithEvents(global.tests.admins.cam2013.client, 1, 5, calendarStart, calendarEnd, function(series) {
+                        var series1 = series[0];
+                        SeriesTestsUtil.generateSerieWithEvents(global.tests.admins.cam2013.client, 1, 5, calendarStart, calendarEnd, function(series) {
+                            var series2 = series[0];
+                            OrgUnitTestsUtil.assertAddOrgUnitSeries(global.tests.admins.cam2013.client, orgUnit.id, [series1.id, series2.id], function() {
+
+                                // Simon subscribes to both series within the context of the organisational unit
+                                SeriesTestsUtil.assertSubscribeSeries(simon.client, series1.id, simon.profile.id, orgUnit.id, function() {
+                                    SeriesTestsUtil.assertSubscribeSeries(simon.client, series2.id, simon.profile.id, orgUnit.id, function() {
+
+                                        // Nico subscribes to both series, but doesn't provide a context
+                                        SeriesTestsUtil.assertSubscribeSeries(nico.client, series1.id, nico.profile.id, null, function() {
+                                            SeriesTestsUtil.assertSubscribeSeries(nico.client, series2.id, nico.profile.id, null, function() {
+
+                                                // Remove series 1 from the organisational unit
+                                                OrgUnitTestsUtil.assertDeleteOrgUnitSeries(global.tests.admins.cam2013.client, orgUnit.id, [series1.id], function() {
+
+                                                    // Verify series 2 has been removed from Simon's calendar
+                                                    UsersTestsUtil.assertGetUserCalendar(simon.client, simon.profile.id, calendarStart, calendarEnd, series2.events, function(calendar) {
+                                                        assert.strictEqual(calendar.results.length, 5);
```

Comment:
> `assert.strictEqual(calendar.results.length, 5);` will assert that Simon's calendar only contains 5 events.

The test for Nico does a `_.union` adding both series1 events and series2 events together to ensure that all 10 events are returned. Which leads me to believe that this might not be reliably ensuring that there are no series1 events.